global.yaml

```diff
@@ -38,7 +38,16 @@ global:
 
   # Default tag for Istio images.
   #tag: release-1.1-latest-daily
-  tag: 1.1.0
+  tag: 1.1.2
+
+  # Comma-separated minimum per-scope logging level of messages to output, in the form of <scope>:<level>,<scope>:<level>
+  # The control plane has different scopes depending on component, but can configure default log level across all components
+  # If empty, default scope and level will be used as configured in code
+  logging:
+    level: "default:info"
+
+  # monitoring port used by mixer, pilot, galley
+  monitoringPort: 15014
```

Comment:
Looking at the original issue (https://github.com/istio/istio/issues/9770) it was because we had a port conflict with Kafka. Since we moved to 15014 we could probably hardcode.

Reply:
I wouldn't call it 'hardcode' - we have the Kustomize 'break glass' to allow people who really need to override ( a bit difficult for CLI args unfortunately - that's why I suggest we start using env variables - like PORT that KNative is using - which are friendlier to Kustomize and installer)